src/oc_erchef/apps/oc_chef_wm/src/chef_reindex.erl

```diff
@@ -252,17 +255,26 @@ log_failures(OrgName, [Failure|Rest]) ->
     lager:error("reindexing[~s] item ~s[~s] failed to reindex: ~s", [OrgName, TypeName, Id, Reason]),
     log_failures(OrgName, Rest).
 
+-spec humanize_failures(list(), list()) -> list().
 humanize_failures([], Acc) ->
     Acc;
 humanize_failures([H|T], Acc) ->
-    {Id, Reason} = H,
-    humanize_failures(T, [{Id, pretty_reason(Reason)} | Acc]).
+    case H of
+        {Id, Reason} -> humanize_failures(T, [{Id, pretty_reason(Reason)} | Acc]);
+        Error -> humanize_failures(T, [{<<"ERROR">>, pretty_reason(Error)} | Acc])
```

Comment:
Since `<<"ERROR">>` is taking the place of Id, maybe `<<"unknown">>` might be better?

Reply:
done :)